run_clang_tidy.sh

```diff
@@ -5,7 +5,14 @@ set -uo pipefail
 exit_code=0
 FILES=$1
 
+EXCLUDED_EXTENSIONS=("pdh", "windows-event-log", "tensorflow")
+
 for changed_file in ${FILES}; do
+  for extension in "${EXCLUDED_EXTENSIONS[@]}"; do
+    if [[ "${changed_file}" =~ extensions/${extension}/ ]]; then
+      continue 2
+    fi
+  done
   if [[ "${changed_file}" == *.cpp ]]; then
     clang-tidy-14 -warnings-as-errors=* -quiet -p build "${changed_file}"
```

Comment:
I have added this as a quick-and dirty way of skipping the Windows-only extensions.  @lordgamez please let me know if you can think of a better way.